### ISSUE TITLE
Guidance on which thermo library to use

### ISSUE THREAD

**Issue by ajuluc** [ReactionMechanismGenerator/RMG-Py#1491]
### Motivation or Problem
There are many thermo libraries to choose from. The documentation shows using primaryThermoLibrary and GRI-Mech as examples, so novice users are led to use those two. It is hard to find any description of the other libraries and when they may be appropriate.

### Desired Solution
A new user will easily find a link to some guidance on which libraries are appropriate when (or at least some way to make an informed choice).

### Potential Alternatives
A link to this page would be a good start: https://rmg.mit.edu/database/thermo/libraries/
but it'd be better if each had a short description (perhaps extracted automatically from the `longDesc` attribute on each library)

**Comment by alongd** [comment]
Related to https://github.com/ReactionMechanismGenerator/RMG-database/issues/191

**Comment by ajuluc** [comment]
I wanted to suggest choosing from the molecule search which thermo libraries best suits the species from the heat capacity estimation plots. That would serve as a good guide in choosing at this link using DME as an example : https://rmg.mit.edu/database/thermo/molecule/1%20C%20u0%20p0%20c0%20%7B2,S%7D%20%7B4,S%7D%20%7B5,S%7D%20%7B6,S%7D%0A2%20O%20u0%20p2%20c0%20%7B1,S%7D%20%7B3,S%7D%0A3%20C%20u0%20p0%20c0%20%7B2,S%7D%20%7B7,S%7D%20%7B8,S%7D%20%7B9,S%7D%0A4%20H%20u0%20p0%20c0%20%7B1,S%7D%0A5%20H%20u0%20p0%20c0%20%7B1,S%7D%0A6%20H%20u0%20p0%20c0%20%7B1,S%7D%0A7%20H%20u0%20p0%20c0%20%7B3,S%7D%0A8%20H%20u0%20p0%20c0%20%7B3,S%7D%0A9%20H%20u0%20p0%20c0%20%7B3,S%7D%0A

**Comment by mjohnson541** [comment]
So recently I started looking at doing this for kinetics libraries:  ReactionMechanIsmGenerator/RMG-database#301

@mliu49 and I discussed this and we think it's best to create a folder for our libraries that are for levels of theory that we automatically use to compile libraries for specific chemistries (such as C3 or C4) based on our ranking of those levels of theory.  We're also currently looking at adding an ATcT specific thermo library.  

I think that having our own compiled seeds with RMG in the name and using them in the examples will probably solve most of this issue.  

We don't currently have an easy way to automatically detect how entries in a given library were determined (although there is usually enough information to determine this manually either from the comments or looking up the paper reference) which really prevents us from automatically knowing whether an entry from CurranPentane is better than an entry from GRI-Mech.  This makes it difficult to rank data that aren't from level of theory/experiment specific libraries.
